circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-    version: 4.6.0
+    version: 6.7.0
```

Comment:
I tried updating the node version as a result of the circleci build failing, obviously, it seems that is not the issue.